### PR TITLE
Set TTL for cluster status and handle it accordingly

### PR DESCRIPTION
**Pull request by shtripat**
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/gluster_integration/sds_sync/cluster_status.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def sync_cluster_status(volumes):
         peer_count=peer_count,
         vol_count=len(volumes),
         volume_up_degraded=degraded_count
-    ).save()
+    ).save(sync_ttl)
```
Comment (nnDarshan) [inline tendrl/gluster_integration/sds_sync/cluster_status.py]: pls pass as keyword argument

Comment (shtripat) [reply tendrl/gluster_integration/sds_sync/cluster_status.py]: Ah yes. Will do that.

**Comment by shtripat** [comment]
Verified the PR with https://github.com/Tendrl/gluster-integration/pull/527 and cluster goes to `unhealthy` state once all the gluster nodes are brought down. Also the brick status fields vanish after TTL under `clusters/{int-id}/Bricks/all/{node-id}/{brick-name}`.

Also in grafana the cluster is marked as `unhealthy` and volume count is set as all zeros with all the hosts marked as `Down`. Once we get the grafana stale time marking it should ideally look fine in grafana dashboard. @cloudbehl we need the grafana PR now.

File: tendrl/gluster_integration/sds_sync/cluster_status.py (hunk 1 above)
Comment (r0h4n) [reply tendrl/gluster_integration/sds_sync/cluster_status.py]: Are you making this change? @shtripat

Comment (shtripat) [reply tendrl/gluster_integration/sds_sync/cluster_status.py]: Ah yes. Looks like didnt push the changes. will do that now

**Pull request merged**